fiona/fio/calc.py

```diff
@@ -0,0 +1,79 @@
+import json
```

Comment:
@perrygeo Need a `from __future__ import division` so we get float division on Python 2.

```console
$ echo '{"properties": {"value": 3}}' | fio calc NEW "f.properties.value / 5"
{"properties": {"NEW": 0, "value": 3}}
```